### DISCUSSION TITLE
jenny_dioco bad pronunciation of British English

### DISCUSSION THREAD

**Discussion by omega3** [rhasspy/piper#356] [Q&A]
I like jenny_dioco model but it has bad pronunciation of British English.
Check out this sample text:

`Negotiate. Union leaders have negotiated an agreement for a shorter working week. Negotiation. The negotiations with the company had reached a crucial stage. Advise. She needed someone to advise her. Advanced. High levels of unemployment in the advanced industrial societies. Advanced learners of English. Advantage. Her experience meant that she had a big advantage over her opponent.`

As reference you can check it out in a dictionary:
https://www.ldoceonline.com/dictionary/negotiate

What can be done to improve it? Do I need to train this model? 
Unfortunately I don't have GPU to process it. Where is the source of this model so that it can be reported?

Other GB models like en_GB-alan-low.onnx seem to have good pronunciation. Although it seems to have some irritating emotion added.

**Comment by Patrick-Ric** [comment]
Maybe this Video from "Thorsten-Voice" can help you:
https://youtu.be/493xbPIQBSU?si=u0ghsJS59wIZnCVh
The tutorial shows you how to improve the pronunciation quality of local text-to-speech services by customising the eSpeak dictionary. Works for ALL local TTS services that use eSpeak or eSpeak-ng as a phonemiser.

**Comment by colbec** [comment]
I just tried to repeat this experiment with the provided text and did not hear a problem, apart from the lack of adequate pauses between what appear to be paragraph breaks at the sentence end periods. It is not a problem to insert such breaks in the text and the subsequent rendering sounds a lot better with adequate silence inserted to make the paragraph breaks more plain. Also playing with sox at 90% speed improves the rendering.

There is a clear (Nothern?) Irish lilt in this voice, Is that confusing the issue at all?

**Comment by omega3** [comment]
Negotiate is pronounced differently as compered to above dictionary link. My [sample](https://drive.google.com/file/d/1w4ugijjjIXE1djaQwpPKvhYGhrCqTKTv/view?usp=drive_link)
But I haven't had time to play with customising the eSpeak dictionary yet.

> (Nothern?) Irish lilt in this voice

Maybe that is the case here. I am not native British so I can't judge.

**Reply by colbec** [reply]
Hmm, it is a tough call, I am accustomed to UK accents and to me the pronunciation of negotiate in this sample is acceptable because it fits with the Irish way of speaking. Any effort to change it to force it to be more acceptable to an unaccustomed ear might make it very odd for listeners used to Irish. So it comes down to your target audience, either force the issue to make it more international or leave as is for a UK listener.

**Comment by omega3** [comment]
Hmm, I didn't realize it was Irish. Now I've found similar pronunciation from British [parliament](https://youtu.be/CacM7JWw0J0?si=OXWoOSiAfKKuKSTU&t=27).
Thank you for this insight. 
I am closing this thread as resolved.
